requirements.txt

```diff
@@ -1,5 +1,6 @@
 -e .
 xmltodict == 0.14.2
+python-openstackclient==7.7.0
```

Comment:
why are we adding this module? We predominately use libcloud to interact with Cloud interfaces

Reply:
Looks like we are using this for quota calcuation.